src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -1,181 +1,400 @@
+                  >
+                    <Select onChange={handleProgramSelect}>
+                      <Option value="">--Choose a Program--</Option>
+                      <Option value={1}>Codercamp</Option>
+                      <Option value={2}>Codersitters</Option>
+                      <Option value={3}>Coderyoga</Option>
+                    </Select>
+                  </Form.Item>
+                  <Form.Item
+                    label={
+                      <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                        Instructor Name:
+                      </label>
+                    }
+                    style={{ width: '65%', fontSize: '1.1rem' }}
+                    onChange={handleChange}
+                  >
+                    <InputNumber
+                      name="instructor_id"
+                      style={{ width: 'auto', fontSize: '1.1rem' }}
+                    />
+                  </Form.Item>
+                </>
+              ) : null}
+              <Form.Item
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Days of the Week:
+                  </label>
+                }
+                valuePropName="checked"
+                style={{ width: '100%', fontSize: '1.1rem' }}
+                onChange={handleCheck}
               >
-                Clear
-              </button>
-            </div>
-          </div>
-          <Card
-            style={{ width: '100%', marginBottom: '7.5%', maxHeight: '100%' }}
-            bodyStyle={{ maxHeight: 100, overflow: 'auto' }}
-          >
-            <p id="prereq-render"></p>
-          </Card>
-
-          <div className="submit-button-container">
-            <button className="submit-button" onClick={handleSubmit}>
-              Submit
-            </button>
-          </div>
-          {/* {props.errorMessage && <div>Error: {props.errorMessage}</div>} */}
-        </Form>
-      </section>
+                <Checkbox.Group
+                  options={[
+                    'Monday',
+                    'Tuesday',
+                    'Wednesday',
+                    'Thursday',
+                    'Friday',
+                    'Saturday',
+                    'Sunday',
+                  ]}
+                />
+              </Form.Item>
+
+              <Form.Item
+                label={
+                  <label style={{ color: '#096A70', fontSize: '1.1rem' }}>
+                    Start and End Time:
+                  </label>
+                }
+                style={{ width: '60%', fontSize: '1.1rem' }}
+              >
+                <TimePicker.RangePicker
+                  onChange={handleTimeChange}
+                  use12Hours={true}
+                  format="HH:mm"
+                  name="course_time"
+                />
+                {/* vvv May be useful for auto-pop vvv */}
+                {/* {props.button_name === 'Add Course'
```

Comment:
Consider removing this commented-out code if it is not being utilized to clean up the code a bit more.